crates/eframe/src/web/text_agent.rs

```diff
@@ -39,6 +44,12 @@ impl TextAgent {
             let input = input.clone();
             move |event: web_sys::InputEvent, runner: &mut AppRunner| {
                 let text = input.value();
+                // Fix android virtual keyboard Gboard
+                // This removes the virtual keyboard's suggestion.
+                if !event.is_composing(){
+                    input.blur().unwrap();
+                    input.focus().unwrap();
```

Comment:
better to not crash on unforseen problems

```suggestion
                    input.blur().ok();
                    input.focus().ok();
```